content/millennium/r4/financial/charge-item.md

```diff
@@ -69,6 +69,47 @@ All URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/r4/St
  `revenue-code`         | None (contains nested extensions) | The type of revenue or cost center providing the product and/or service.
  `unit-price`           | [`Money`]                         | The price of a single unit for the resource.
 
+## Search
+
+<%= beta_tag(action: true) %>
+
+Search for ChargeItems that meet supplied query parameters:
+
+    GET /ChargeItem?:parameters
+
+### Authorization Types
+
+<%= authorization_types(provider: true, patient: false, system: true) %>
+
+### Parameters
+
+ Name                  | Required?  | Type          | Description
+-----------------------|------------|---------------|-------------------------------------------------------
+ `_id`                 | Yes        | [`token`]     | The logical resource id associated with the ChargeItem.
+
+Notes:
+
+- When searching by `_id`, only a single ChargeItem id can be provided.
+
+### Headers
+
+<%= headers %>
+
+### Example - Search by Id
+
+#### Request
+
+    GET https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/ChargeItem?_id=292870306
+
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:r4_chargeitem_bundle) %>
+
```

Comment:
We should have the non-normative disclaimer after the search, the same as retrieve.

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/653/commits/ac94231f91a74c9951c3412029474c604195b416#diff-16d4f8f74626c62d57a8b71d6be699f60e72fffed8636bc32e6654d4252b3ebbR109-R110